src/service/index.ts

```diff
@@ -1,10 +1,27 @@
-import { socketClient } from 'src/socket';
+import { socketClient } from 'src/service/socket';
 import { blockchainInfoRepository } from 'src/repository/blockchainInfo';
 import { blockRepository } from 'src/repository/block';
 import { BlockchainInfoService } from 'src/service/blockchainInfo';
 import { BlockService } from 'src/service/block';
 import { AccountService } from 'src/service/account';
+import { WebhookService, WebhookAction } from 'src/service/webhook';
+import { ON_APPLY_TRANSACTION, ON_APPLY_BLOCK } from 'src/config';
+import { EVENT_TYPES } from 'ddk.registry/dist/model/transport/event';
 
 export const blockService = new BlockService(blockRepository, socketClient);
 export const blockchainInfoService = new BlockchainInfoService(blockchainInfoRepository, socketClient);
 export const accountService = new AccountService();
+export const webhookService = new WebhookService<WebhookAction | EVENT_TYPES>();
+
+if (ON_APPLY_TRANSACTION) {
```

Comment:
how about to extract this to some webhook init file? in this case service/index will have better look